source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/ExplicitDimensionTest.cpp

```diff
@@ -455,6 +455,12 @@ namespace AdaptiveCardsSharedModelUnitTest
                     {\
                         \"type\":\"ColumnSet\",\
                         \"columns\": [\
+                            {\
```

Comment:
since this is positive test; i added additional column to make the payload valid payload.